### ISSUE TITLE
Feedback button is button inside a link...

### ISSUE THREAD

**Issue by dominikwilkowski** [govau/service-manual#360]
Please make it just a link and remove the button. Let me know if there are issues with that. Right now that is an a11y fail as you tab twice through it.

![screen shot 2017-09-20 at 4 11 05 pm](https://user-images.githubusercontent.com/1266923/30629276-529d7824-9e1e-11e7-8322-07f9cdb58c4b.png)
